test/e2e/iam/aws_iam_policy.jsonc

```diff
@@ -129,8 +132,7 @@
             "Action": [
                 "sqs:GetQueueUrl",
                 "sqs:ReceiveMessage",
-                "sqs:DeleteMessage",
-                "sqs:DeleteMessageBatch"
```

Comment:
AWS was complaining that this permission doesn't exist, so I removed it here too.